hw/ip/alert_handler/doc/dv_plan/index.md

```diff
@@ -67,25 +69,34 @@ The `alert_handler_base_vseq` virtual sequence is extended from `cip_base_vseq`
 All test sequences are extended from `alert_handler_base_vseq`.
 It provides commonly used handles, variables, functions and tasks that the test sequences can simple use / call.
 Some of the most commonly used tasks / functions are as follows:
-// Work in Process:
-* task 1:
-* task 2:
+* drive_alert:     Drive alert_tx signal pairs through `alert_esc_if` interface
+* read_ecs_status: Readout registers that reflect escalation status, including `classa/b/c/d_accum_cnt`, `classa/b/c/d_esc_cnt`, and `classa/b/c/d_state`
 
 #### Functional coverage
 To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
 The following covergroups have been developed to prove that the test intent has been adequately met:
-// Work in Process:
-* cg1:
-* cg2:
+* accum_cnt_cg:      Cover number of alerts triggered under the same class
+* esc_sig_length_cg: Cover signal length of each escalation pairs
 
 ### Self-checking strategy
 #### Scoreboard
 The `alert_handler_scoreboard` is primarily used for end to end checking.
 It creates the following analysis ports to retrieve the data monitored by corresponding interface agents:
-// Work in Process:
 * tl_a_chan_fifo: tl address channel
 * tl_d_chan_fifo: tl data channel
-<!-- explain inputs monitored, flow of data and outputs checked -->
+* alert_fifo:     An array of `alert_fifo` that connects to corresponding alert_monitors
+* esc_fifo:       An array of `esc_fifo` that connects to corresponding esc_monitors
+
+Alert_handler scoreboard monitors all valid CSR registers, alert handshakes, and escalation handshakes.
+To ensure certain alert, interrupt, or escalation signals are triggered at the expected time, the alert_handler scoreboard implemented a few counters:
+* intr_cnter_per_class[NUM_ALERT_HANDLER_CLASSES]: Count number of clock cycles that the interrupt bit stays high.
+  If the stored number is larger than the `timeout_cyc` registers, the corresponding escalation is expected to be triggered
+* accum_cnter_per_class[NUM_ALERT_HANDLER_CLASSES]: Count number of alerts triggered under the same class.
+  If the stored number is larger than the `accum_threshold` registers, the corresponding escalation is expected to be triggered
+* esc_cnter_per_signal[NUM_ESC_SIGNALS]: Count number of clock cycles that each escalation signals stays high.
```

Comment:
nit: signals -> signal

Reply:
Thanks Weicai, updated as you suggested.